### PR TITLE
Saddle new brand

### PR DESCRIPTION
**Pull request by nkrulikovsky**

### CODE REVIEW

File: src/theme/palette.ts [hunk 1]
```diff
@@ -215,14 +202,18 @@ const darkPalette: PaletteOptions | undefined = {
...
+    default: "#181818",
+    paper: "#303030",
   },
-  divider: "#311188",
+  divider: "#7272BA",
   other: {
-    divider: "#311188",
-    border: "#7D7D7D",
+    divider: "#7272BA",
+    border: "#474799              ",
```
Comment (hammeiam) [inline src/theme/palette.ts:211]: rm extra spaces

File: src/theme/palette.ts [hunk 2]
```diff
@@ -1,18 +1,36 @@
...
+  if (color1 && color2)
+    return `linear-gradient(90deg, ${color1} 0%, ${color2} 100%)`
+}
+
+// function createGradient3(color1:string,color2:string,color3:string){
+
+// }
+
+type GradientsPaletteOptions = {
+  primaryLight?: string
```
Comment (hammeiam) [inline src/theme/palette.ts:17]: Should all of these types be optional?

File: src/theme/palette.ts [hunk 3]
```diff
@@ -1,18 +1,36 @@
+import { OriginColorVariant } from "../types"
 import { PaletteOptions } from "@mui/material"
 
 // Define custom color types
 
+export function createGradient2(color1?: string, color2?: string) {
+  console.log("color 1==?", color1, "color 2 ==>", color2)
```
Comment (hammeiam) [inline src/theme/palette.ts:7]: rm console log

File: src/theme/palette.ts [hunk 4]
```diff
@@ -1,18 +1,36 @@
...
+  if (color1 && color2)
+    return `linear-gradient(90deg, ${color1} 0%, ${color2} 100%)`
+}
+
+// function createGradient3(color1:string,color2:string,color3:string){
```
Comment (hammeiam) [inline src/theme/palette.ts:12]: rm comment

File: src/pages/Farm/FarmOverview.tsx [hunk 5]
```diff
@@ -59,7 +59,7 @@ FarmOverviewProps): JSX.Element | null {
       direction="row"
       sx={{
         backgroundColor: (theme) => theme.palette.background.paper,
-        border: (theme) => `${theme.palette.other.divider} solid 1px`,
+        // border: (theme) => `${theme.palette.other.divider} solid 1px`,
```
Comment (hammeiam) [inline src/pages/Farm/FarmOverview.tsx:62]: rm comment

File: src/components/SwapTokenInput.tsx [hunk 6]
```diff
@@ -122,19 +122,20 @@ export default function SwapTokenInput({
...
+            <Typography variant="subtitle1" color="secondary.light">
               {selectedTokenSymbol || "Choose"}
             </Typography>
           </Button>
           <Typography
             variant="body2"
             noWrap
             paddingLeft={1}
-            color="text.secondary"
+            color="text.secondary."
```
Comment (hammeiam) [inline src/components/SwapTokenInput.tsx:138]: rm period

**Review by hammeiam — COMMENTED** [review]
Welcome back Nikita! This all looks great minus some minor nitpicks. I'm also having trouble viewing the build preview, I'm not sure what the issue is there

File: src/theme/palette.ts (hunk 1 above)
Comment (nkrulikovsky) [reply src/theme/palette.ts]: remoted

File: src/pages/Farm/FarmOverview.tsx (hunk 5 above)
Comment (nkrulikovsky) [reply src/pages/Farm/FarmOverview.tsx]: ✅

File: src/theme/palette.ts (hunk 4 above)
Comment (nkrulikovsky) [reply src/theme/palette.ts]: ✅

File: src/components/SwapTokenInput.tsx (hunk 6 above)
Comment (nkrulikovsky) [reply src/components/SwapTokenInput.tsx]: ✅

File: src/theme/palette.ts (hunk 2 above)
Comment (nkrulikovsky) [reply src/theme/palette.ts:12]: Don't need to be optional.
Actually, we defined most of gradient colors but didn't use much on design. So I removed most of theme only give `primary`, `secondary`, `gold`, `logo`.
We can give more theme colors when we'll need.

**Review by hammeiam — APPROVED** [review]

**Pull request merged**